src/Aspire.Dashboard/Components/ResourcesGridColumns/StartTimeColumnDisplay.razor

```diff
@@ -0,0 +1,23 @@
+﻿@using Aspire.Dashboard.Model
+@using Aspire.Dashboard.Resources
+
+@if (Resource.CreationTimeStamp is not null)
+{
+    var localTime = Resource.CreationTimeStamp.Value.ToLocalTime();
+
+    if (localTime.Date == DateTime.Now.Date)
+    {
+        // e.g. "08:57" (based on user's culture and preferences)
+        @localTime.ToShortTimeString()
+    }
+    else
+    {
+        // e.g. "9/02/2024 08:57:44" (based on user's culture and preferences)
+        @localTime.ToString()
```

Comment:
```suggestion
        @localTime.ToString(CultureInfo.CurrentCulture)
```
nit: Make it obvious the culture is current. An analyzer would prompt for this in the razor.cs file, but they don't run in razor files.